src/utilities/transition/BaseTransition.js

```diff
@@ -0,0 +1,250 @@
+    } else {
+      this.trigger( BaseTransition.BEGIN_EVENT, { target: this } );
+      _transitionCompleteBinded();
+    }
+  }
+
+  /**
+   * Remove an event listener to the transition.
+   */
+  function _removeEventListener() {
+    _dom.removeEventListener( _transitionEndEvent, _transitionCompleteBinded );
+  }
+
+  /**
+   * Handle the end of a transition.
+   */
+  function _transitionComplete() {
+    _removeEventListener();
+    this.trigger( BaseTransition.END_EVENT, { target: this } );
+    _isAnimating = false;
+  }
+
+  /**
+   * Search for and remove initial BaseTransition classes that have
+   * already been applied to this BaseTransition's target element.
+   */
+  function _flush() {
+    for ( var prop in _classes ) {
+      if ( _classes.hasOwnProperty( prop ) &&
+           _classes[prop] !== _classes.BASE_CLASS &&
+           _dom.classList.contains( _classes[prop] ) ) {
+        _dom.classList.remove( _classes[prop] );
+      }
+    }
+  }
+
+  /**
+   * Remove all transition classes, if transition is initialized.
+   * @returns {boolean}
+   *   True, if the element's CSS classes were touched, false otherwise.
+   */
+  function remove() {
+    if ( _dom ) {
+      halt();
+      _dom.classList.remove( _classes.BASE_CLASS );
+      _flush();
+      return true;
+    }
+
+    return false;
+  }
+
+  /**
+   * @param {string} className - A CSS class.
+   * @returns {boolean} False if the class is already applied
+   *   or the transition is not initialized,
+   *   otherwise true if the class was applied.
+   */
+  function applyClass( className ) {
+    if ( !_dom ) { return false; }
+    if ( !_isFlushed ) {
+      _flush();
+      _isFlushed = true;
+    }
+
+    if ( _dom.classList.contains( className ) ) {
+      return false;
+    }
+
+    _removeEventListener();
+    _dom.classList.remove( _lastClass );
+    _lastClass = className;
+    _addEventListenerBinded();
+    _dom.classList.add( _lastClass );
+
+    return true;
+  }
+
+  // TODO: Update Expandables to use a transition.
+  /**
+   * @param {HTMLNode} elem
+   *   The element to check for support of transition end event.
+   * @returns {string} The browser-prefixed transition end event.
+   */
+  function _getTransitionEndEvent( elem ) {
+    if ( !elem ) {
+      var msg = 'Element does not have TransitionEnd event. It may be null!';
+      throw new Error( msg );
+    }
+
+    var transition;
+    var transitions = {
+      WebkitTransition: 'webkitTransitionEnd',
+      MozTransition:    'transitionend',
+      OTransition:      'oTransitionEnd otransitionend',
+      transition:       'transitionend'
+    };
+
+    for ( var t in transitions ) {
+      if ( transitions.hasOwnProperty( t ) &&
+           typeof elem.style[t] !== 'undefined' ) {
+        transition = transitions[t];
+        break;
+      }
+    }
+    return transition;
+  }
+
+  // Attach public events.
+  var eventObserver = new EventObserver();
```

Comment:
This is working for me locally because of a mishap in my workflow (I'm editing `AtomicComponent` directly inside the `node_modules` of `capital_framework` to test changes, then copying those changes to my working repo of `AtomicComponent`, and I think I just forgot to copy the changes to `BaseTransition.js` - sorry!) and will be fixed in my next commit.